### DISCUSSION TITLE
Please help me out in installing dlib package

### DISCUSSION THREAD

**Discussion by sisya-star** [datasciencecompleteguide/Machine-Learning#34] [Q&A]
I watche dyour youtube video and got an error while installing dlib so i came here for solutions ,So please help me out
![Screenshot (91)](https://user-images.githubusercontent.com/57221607/153143058-bbc737e9-b416-4665-a5bd-789ae96e0e50.png)
![Screenshot (92)](https://user-images.githubusercontent.com/57221607/153143071-6f597724-6750-4db7-8b69-57d8f2af2b51.png)

**Comment by datasciencecompleteguide** [comment]
Hey,
You should use the lower version of Python instead of using 3.10
I would recommend you python 3.8.6

**Comment by sisya-star** [comment]
Yes Sir
i installed 3.6.0
Is that enough Sir ?
